### DISCUSSION TITLE
File is not uploading in ame wizard

### DISCUSSION THREAD

**Discussion by mesadsadx** [Atlas-OS/Atlas#1296] [Help & Support]
![image](https://github.com/user-attachments/assets/d9756dbe-0178-4068-9e8b-de54530f4a3e)
the file is not uploaded to ame wizard (ame is not downloaded from the official site) gives an error (error 27, 39 is present in the xml document) windows 24h2, without a microsoft account

**Comment by LumiFae** [comment]
Hi, we do not do support with the AME wizard, but by what you are saying it may be because you didn't download the official version, please download it from their site.

**Comment by pencilnav** [comment]
Your AME wizard is outdated. Please use this link to download the latest version from Ameliorated.
https://download.ameliorated.io/AME%20Wizard%20Beta.zip
